minikube/secret.yaml

```diff
@@ -0,0 +1,19 @@
+# secret.yaml
+#
+# To encode the secret value use
+#   echo -n "secret value" |base64 -w0
+#
+apiVersion: v1
+kind: Secret
+metadata:
+  name: module-14
+type: Opaque
+data:
+  MONGO_URL: <encoded DB connection string>
```

Comment:
So that no one ever accidentally commits the secrets, we would usually follow the same pattern as the .env file. Have a committed secret.yaml.template that looks like this, the documentation says to copy it to secret.yaml and fill it in, and .gitignore contains secret.yaml